### DISCUSSION TITLE
No sku returned in Play Store Production bundle.

### DISCUSSION THREAD

**Discussion by Cluster-Taek** [hyochan/react-native-iap#1691] [Q&A] [answered]
Hi, I want to know the reason for this issue.

I used this library for my subscription system.

I tested my sub system and get this result.

O : can use in app purchase
X : can't use in app purchase
>> On Debug APK : Testing User (O), Other User (X) 
>> On Release APK : Testing User (O)
>> On Play Store : Someone Play Store Google Account (O), Someone Play Store Google Account (X)

Some Google accounts were able to pay in-app, while others failed to pay in-app in "Same Application".

I checked Failed Log and I find this log. No sku returned in Play Store Production bundle.
<b>ReactNativeJS: [Error: The sku was not found. Please fetch products first by calling getItems]</b>

There's no pattern in this issue, so I can't find what is reason.
Also I checked play console and there's no problem.

Does anyone else have the same problem?
Please someone help me. Thank you.

**Comment by Cluster-Taek** [comment]
I solved this problem, anyway thank you!

**Comment by vtoupet** [comment, marked as answer]
> I solved this problem, anyway thank you!

@Cluster-Taek Could you tell us how you solved this issue ? It might help others (like me for instance ;-))

**Reply by Cluster-Taek** [reply]
My server was the problem, not the library 😂
There was a problem because an issue occurred at the end point where the product ID value was loaded.
